R/data.R

```diff
@@ -52,3 +52,68 @@
+#'   \item{identified_name}{Taxon name as it appears in the original publication,
+#'   which may include expressions of uncertainty (e.g. "cf.", "aff.", "?") or
+#'   novelty (e.g. "n. gen.", "n. sp.")}
+#'   \item{identified_rank}{The taxonomic rank, or resolution, of the identified
+#'   name}
+#'   \item{accepted_name}{Taxon name once the identified name has passed through
+#'   the Paleobiology Database's internal taxonomy, which collapses synonyms,
+#'   amends binomials which have been altered (e.g. species moving to another
+#'   genus) and updates taxa which are no longer valid (e.g. _nomina dubia_)}
+#'   \item{accepted_rank}{The taxonomic rank, or resolution, of the accepted
+#'   name}
+#'   \item{early_interval}{The oldest (or only) time interval within which the
+#'   occurrence is thought to have been deposited}
+#'   \item{late_interval}{The youngest time interval within which the occurrence
+#'   is thought to have been deposited}
+#'   \item{max_ma, min_ma}{The age range given to the occurrence}
+#'   \item{phylum, class, order, family, genus}{The taxa (of decreasing taxonomic
+#'   level) which the occurrence is identified as belonging to}
+#'   \item{abund_value, abund_unit}{The number (and units) of fossils attributed
+#'   to the occurrence}
+#'   \item{lng, lat}{The modern-day longitude and latitude of the fossil locality}
+#'   \item{collection_name}{The name of the Paleobiology Database collection
+#'   which the occurrence belongs to, typically a spatio-temporally restricted
+#'   locality}
+#'   \item{cc}{The country (code) where the fossils were discovered}
+#'   \item{paleolng, paleolat}{The longitude and latitude at which the fossils
+#'   were deposited, estimated by rotating the modern-day co-ordinates back to
+#'   the age of the collection using a global plate model (see
+#'   \url{https://paleobiodb.org/} for more details)}
+#'   \item{formation, stratgroup, member}{The geological units from which the
+#'   fossils were collected}
+#'   \item{zone}{The biozone which the occurrence is attributed to}
+#'   \item{lithology1}{The main lithology of the beds in the section where the
+#'   fossils were collected}
+#'   \item{environment}{The inferred environmental conditions in the place of
+#'   deposition}
+#'   \item{pres_mode}{The mode of preservation of the fossils found in the
+#'   collection (not necessarily of that specific occurrence), which will include
+#'   information on whether they are body or trace fossils}
+#'   \item{taxon_environment}{The environment within which the taxon is thought
+#'   to have lived, collated within the Paleobiology Database}
+#'   \item{motility, life_habit, diet}{Various types of trait data for the taxon,
+#'   collated within the Paleobiology Database}
+
+#' }
+#' @section References:
+#' Uhen MD et al. (in prep). Paleobiology Database User Guide.
+#' \cr
+#' @source Compiled by Bethany Allen. See item descriptions for details.
+"early_tetrapods"
```

Comment:
I can't seem to call the help for this documentation i.e. ?early_tetrapods. It looks like you still need to build the .Rd file. This can be done using devtools::document()